pkgs/data/fonts/culmus/default.nix

```diff
@@ -3,17 +3,24 @@
 let
   version = "0.133";
 in fetchzip {
-  name = "culmus-${version}";
+  name = "culmus-${version}-1";
 
   url = "mirror://sourceforge/culmus/culmus/${version}/culmus-${version}.tar.gz";
 
   postFetch = ''
     tar -xzvf $downloadedFile --strip-components=1
-    mkdir -p $out/share/fonts/truetype
+    mkdir -p $out/share/fonts/{truetype,type1}
+    cp -v *.pfa $out/share/fonts/type1/
+    cp -v *.afm $out/share/fonts/type1/
+    cp -v fonts.scale-type1 $out/share/fonts/type1/fonts.scale
     cp -v *.ttf $out/share/fonts/truetype/
+    cp -v *.otf $out/share/fonts/truetype/
+    cp -v fonts.scale-ttf $out/share/fonts/truetype/fonts.scale
+    mkdir -p $out/etc/fonts/conf.d
```

Comment:
In NixOS the etc dir of package has no effect. Is this really needed?

Reply:
No, I just translated the installation script.